test/test-config.sh

```diff
@@ -43,6 +43,10 @@ fi
 # This is needed for "make push-images".
 : ${TEST_BUILD_PMEM_REGISTRY:=localhost:5000}
 
+# The PMEM-CSI driver/operator container image tag use.
+# This is used by version skew tests to set the driver/operator version.
+: ${TEST_PMEM_IMAGE_TAG:=canary}
```

Comment:
How will this work once we have this test in a release-1.0 branch and test against 0.9? We cannot use "canary" as the image in such a branch.

Reply:
The `hack/set-version.sh` might have to be extended to replace the "canary" version also in this line.

Reply:
> The `hack/set-version.sh` might have to be extended to replace the "canary" version also in this line.

Done, Also removed `canary` from the regex in `test/start-operator.sh`.